.github/workflows/pr.yml

```diff
@@ -46,7 +46,7 @@ jobs:
           service_account_key: ${{ secrets.GCP_SA_KEY }}
           export_default_credentials: true
       - name: Install dependencies
-        run: make install
+        run: make install && pip install pytest-timeout
```

Comment:
Can we add this to setup.py, ideally as a dev dependency?

Reply:
Done, I borrowed it from policyengine-core.
It is a bit confusing (maybe just for me) because it relies on the the makefile `install` rule to be used only for development, but it I think it is the case.

Reply:
Sorry just one more minor comment on this: do we need to pin to the patch version? No problem if we do but otherwise pinning to major (preferable) or minor versions would help lower the risk of dependency conflicts later down the line.

Reply:
OK, I removed the version, I don't think we need it.